sources/Cargo.toml

```
@@ -37,6 +38,7 @@ members = [
     "api/migration/migrations/v1.1.2/control-container-v0-5-1",
     "api/migration/migrations/v1.1.3/kubelet-cpu-manager-state",
     "api/migration/migrations/v1.1.3/kubelet-cpu-manager",
+    "api/migration/migrations/v1.1.4/add-custom-certificates",
```

Looks like the version here doesn't match `Release.toml`.

```suggestion
    "api/migration/migrations/v1.2.0/add-custom-certificates",
```